fdbserver/BlobManifest.actor.cpp

```diff
@@ -402,11 +403,7 @@ class BlobManifestDumper : public ReferenceCounted<BlobManifestDumper> {
 						if (!result.more) {
 							break;
 						}
-						if (result.readThrough.present()) {
-							begin = firstGreaterOrEqual(result.readThrough.get());
-						} else {
-							begin = firstGreaterThan(result.back().key);
-						}
+						begin = firstGreaterOrEqual(result.getReadThrough(beginKeyArena));
```

Comment:
This introduces a copy when read through isn't set. I'm not sure how concerning that is.